tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "rootDir": "./src/",
+    "outDir": "./lib/",
+    "sourceMap": true,
+    "types": ["node"],
+    "target": "es6",
+    "noImplicitAny": true,
+    "strictNullChecks": true
+  },
+  "exclude": [
+    "node_modules",
+    "lib",
+    "tests"
+  ]
+}
```

Comment:
I'm hesitant to commit to that yet, without a tslint configuration decided. As soon as the coding standards etc have settled I'll be happy to add it in and make the required changes.